securedrop/dockerfiles/focal/python3/Dockerfile

```diff
@@ -42,14 +44,17 @@ RUN curl -s https://archive.mozilla.org/pub/firefox/releases/${FF_VERSION}/KEY |
 # Install the version of Firefox on which Tor Browser is based
 RUN curl -LO https://archive.mozilla.org/pub/firefox/releases/${FF_VERSION}/linux-x86_64/en-US/firefox-${FF_VERSION}.tar.bz2 && \
     curl -LO https://archive.mozilla.org/pub/firefox/releases/${FF_VERSION}/linux-x86_64/en-US/firefox-${FF_VERSION}.tar.bz2.asc && \
-    gpg2 --verify firefox-${FF_VERSION}.tar.bz2.asc 2>&1 | grep "Primary key fingerprint:" | sed -e 's/Primary key fingerprint: //' -e 's/ //g' | tail -1 | grep -qE "${MOZILLA_RELEASE_KEY_FINGERPRINT}" && \
+    gpg2 --keyring ./firefox.keyring --export ${MOZILLA_RELEASE_KEY_FINGERPRINT} && \
+    gpgv --keyring ./firefox.keyring ~/Downloads/firefox-${FF_VERSION}.tar.bz2.asc ~/Downloads/firefox-${FF_VERSION}.tar.bz2 && \
     tar xjf firefox-*.tar.bz2 && \
     mv firefox /usr/bin
 
 # Install geckodriver
 RUN wget https://github.com/mozilla/geckodriver/releases/download/${GECKODRIVER_VERSION}/geckodriver-${GECKODRIVER_VERSION}-linux64.tar.gz && \
     wget https://github.com/mozilla/geckodriver/releases/download/${GECKODRIVER_VERSION}/geckodriver-${GECKODRIVER_VERSION}-linux64.tar.gz.asc && \
-    gpg2 --verify geckodriver-${GECKODRIVER_VERSION}-linux64.tar.gz.asc && \
+    # geckodriver uses the same key used by Mozilla
+    gpg2 --keyring ./geckodriver.keyring --export ${MOZILLA_RELEASE_KEY_FINGERPRINT} && \
```

Comment:
Instead of re-exporting a new one for geckodriver, we can just use the same `firefox.keyring`. Maybe name it `mozilla.keyring`?

Big +1 to adding the comment, that's a great idea.